apps/server/src/modules/oauth/service/oauth.service.ts

```diff
@@ -102,13 +104,15 @@ export class OAuthService {
 		return verifiedJWT as IJwt;
 	}
 
-	async findUser(decodedJwt: IJwt, system: System): Promise<User> {
-		// iserv strategy
-		if (system.oauthConfig && system.oauthConfig.provider === 'iserv') {
-			return this.iservOauthService.findUserById(system.id, decodedJwt);
-		}
+	async findUser(accessToken: string, decodedJwt: IJwt, system: System): Promise<User> {
 		try {
-			const user = await this.userRepo.findById(decodedJwt.sub);
+			// iserv strategy
+			if (system.oauthConfig && system.oauthConfig.provider === 'iserv') {
+				return await this.iservOauthService.findUserById(system.id, decodedJwt);
+			}
+			this.logger.debug('provisioning is running...');
```

Comment:
Which provisioning? Maybe you can log some meta data. This log would bring nothing in an analysis